src/parse/package.rs

```diff
@@ -113,15 +115,15 @@ pub fn parse_open<'a>(
     let (i, name) = parse_name(i)?;
     let (i, _) = parse_space(i)?;
     let (i, alias) = opt(terminated(parse_alias, parse_space))(i)?;
-    let alias = alias.unwrap_or(String::from(""));
+    let alias = alias.unwrap_or_else(|| String::from(""));
```

Comment:
This seems unnecessary, we don't need the closure.

Reply:
The way the code was before, there would be an allocation even if `alias` was `Ok`. `unwrap_or_else` makes it conditional upon `alias` being `Err`, removing an unnecesary allocation in the happy path.

Reply:
Oh, I see, this is one of those of those Rust idioms my FP brain just skipped over :smile: . Makes sense!